steps/dns/steps.go

```diff
@@ -40,7 +40,12 @@ func (s Steps) InitializeSteps(ctx context.Context, scenCtx *godog.ScenarioConte
 
 	// Initialize the steps
 	scenCtx.Step(`^the DNS server "([^"]*)"$`, func(svr string) error {
-		return session.ConfigureServer(ctx, golium.ValueAsString(ctx, svr))
+		// DNS transport protocol is set to UDP by default
+		var transport = "UDP"
+		return session.ConfigureServer(ctx, golium.ValueAsString(ctx, svr), golium.ValueAsString(ctx, transport))
```

Comment:
it does not make sense to process the transport with ValueAsString.

Reply:
Right. Done